vendor-extra/LiberoContentBundle/src/ViewConverter/BoldVisitor.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Libero\LiberoContentBundle\ViewConverter;
+
+use FluentDOM\DOM\Element;
+use Libero\ViewsBundle\Views\ConvertsChildren;
+use Libero\ViewsBundle\Views\SimplifiedChildVisitor;
+use Libero\ViewsBundle\Views\View;
+use Libero\ViewsBundle\Views\ViewConverter;
+use Libero\ViewsBundle\Views\ViewConverterVisitor;
+
+final class BoldVisitor implements ViewConverterVisitor
+{
+    use ConvertsChildren;
+    use SimplifiedChildVisitor;
+
+    private $converter;
+
+    public function __construct(ViewConverter $converter)
+    {
+        $this->converter = $converter;
+    }
+
+    protected function doVisit(Element $object, View $view, array &$context = []) : View
+    {
+        return $view->withArgument('text', $this->convertChildren($object, $context));
+    }
+
+    protected function possibleTemplate() : string
+    {
+        return '@LiberoPatterns/bold.html.twig';
+    }
+
+    protected function expectedElement() : string
+    {
+        return '{http://libero.pub}b';
```

Comment:
We'll need to add a schema for this, but it doesn't exactly require thought so we can do it now.

Reply:
Added to the checklist on https://github.com/libero/libero/issues/104.

Reply:
https://github.com/libero/schemas/pull/15